pipeline_steps/common.groovy

```diff
@@ -1954,6 +1962,52 @@ void createRelease(String component_text, Boolean from_rc_branch){
   )
 }
 
+void updateDependents(String componentName){
+  def parallelBuilds = [:]
+  List allWorkflowJobs = Hudson.instance.getAllItems(WorkflowJob)
+  venv = "${WORKSPACE}/.componentvenv"
+
+  sh """#!/bin/bash -xe
+    if [ ! -e ${venv} ]; then
+      virtualenv --python python3 ${venv}
+      set +x; . ${venv}/bin/activate; set -x
+      pip install -c '${env.WORKSPACE}/rpc-gating/constraints_rpc_component.txt' rpc_component
+    fi
+  """
+
+  def dependentComponents_text = sh(
+    returnStdout: true,
+    script: """#!/bin/bash
+      set +x; . ${venv}/bin/activate; set -x
+      component --releases-dir ./ dependents --component-name ${componentName} get
+    """
+  )
+
+  def dependentComponents = readYaml text: dependentComponents_text
+  def dependentComponentsToUpdate = dependentComponents["name"]
+
+  for ( dependentComponent in dependentComponentsToUpdate ) {
+    jobs = (allWorkflowJobs.findAll { it.displayName =~ /PM-Dep-Update_${dependentComponent}.*/ })
+    for ( job in jobs ) {
+      parallelBuilds[job.displayName] = {
```

Comment:
There isn't much point in building a parallel map, when `wait: false` is set on the builds. May as well kick them off in a loop.